imap_processing/codice/constants.py

```diff
@@ -1,10 +1,11 @@
-"""Contains constants variables to support CoDICE processing.
+"""
+Contains constants variables to support CoDICE processing.
 
 The ``plan_id``, ``plan_step``, and ``view_id`` mentioned in this module are
 derived from the packet data.
 
-Acronyms
---------
```

Comment:
I thought I was being cool by adding an `Acronyms` section. But `numpydoc` is not a fan 😞 😅